docs/tutorials/send-payjoin-with-pdk.md

```diff
@@ -0,0 +1,403 @@
+# Introduction
+
+This tutorial goal is to get you started with payjoin and demonstrate
+payjoin capabilities. We will demonstrate how to setup the sender side
+in a payjoin transaction and transact with a payjoin receiver
+utilising `payjoin-cli`.
+
+While `rust-payjoin` supports both BiP0078 and BiP0077, this tutorial
+is specificly to demonstrate BiP0078(payjoin version 1).
+
+For a quick wallet setup, we utilise `docker` and `bitcoincore_rpc`
+crate to run a regtest node and fund two wallets, one for the sender
+and one for the receiver.
+
+For any feedback, please free to reach out through the payjoin
+official discord or github.
+
+# 1. Bitcoin Node
+ The following command will start a bitcoin core node in regtest mode,
+ with auth setup already in place just for testing purposes.
+
+ *You would need to have docker installed in order to run the
+ following command*
+
+
+ ```
+ docker run --name bitcoin-server -d --rm -it \
+ -p 18443:18443 \
+ -p 18444:18444 \
+ ruimarinho/bitcoin-core \
+ -printtoconsole \
+ -regtest=1 \
+ -rpcallowip=0.0.0.0/0 \
+ -rpcbind=0.0.0.0 \
+ -rpcauth='foo:7d9ba5ae63c3d4dc30583ff4fe65a67e$9e3634e81c11659e3de036d0bf88f89cd169c1039e6e09607562d54765c649cc'
+ ```
```

Comment:
Docker might be able to reckon with to write a tutorial, but it's more work for the person following the tutorial and leaves side-effects on their system. I'd much rather we use the [`bitcoind` crate](https://docs.rs/bitcoind/latest/bitcoind/) to handle that setup and tear down for them.